src/ansi-c/library/gcc.c

```diff
@@ -32,3 +32,111 @@ inline void __sync_synchronize(void)
                   "WWcumul", "RRcumul", "RWcumul", "WRcumul");
   #endif
 }
+
+/* FUNCTION: __atomic_test_and_set */
+
+void __atomic_thread_fence(int memorder);
```

Comment:
Prototyped this twice?

Reply:
Yes, intentionally: each of the `FUNCTION:` sections is parsed individually, and thus needs to have all the necessary prototypes.